src/framework/uicomponents/qml/MuseScore/UiComponents/StyledDialogView.qml

```diff
@@ -51,6 +51,9 @@ DialogView {
         accessibilityActiveTimer.start()
     }
 
+    onWidthChanged: root.updateGeometry()
+    onHeightChanged: root.updateGeometry()
```

Comment:
If only it was that easy 😉

This causes problems for resizable dialogs: as soon as you try to resize them, they jump back to their initial position and size, making it impossible to resize them.


https://github.com/musescore/MuseScore/assets/48658420/c78ee1fb-7a20-4022-851f-d4532e4c9925

Reply:
Can you explain why you need to call geometry update on the size change? What problem are you solving here?

Reply:
I think I might know why: for some dialogs, when they are opened, their "layout" is not fully computed. So the geometry used for centering the dialog, is not the definitive geometry. After the dialog has appeared, its layout is finalised, so the geometry changes (the dialog becomes taller, in most cases). The result is that the dialog is not centered anymore.
This applies at least to the "standard dialogs", but apparently also for the scanning dialog. 

Is that right, @HemantAntony?